pkg/apiserver/handlers.go

```diff
@@ -534,3 +534,28 @@ func (r *APIRequestInfoResolver) GetAPIRequestInfo(req *http.Request) (APIReques
 
 	return requestInfo, nil
 }
+
+// IsAPIRequest looks for patterns like /prefix/version/resource and /prefix/group/version/resource. It if matches, returns true.
```

Comment:
I'm in favor of updating that function rather than adding this function. If the two marked places aren't errors, they should perhaps set something in the request info (e.g., HasGroup/Version/Resource or make those fields pointers or something). But I'm also OK with returning different errors. I think just declaring an ErrNotAPIRequest and ErrMalformedAPIRequest are sufficient, right?